doc/NEWS

```diff
@@ -186,6 +186,8 @@ Major changes from 1.9.0-jumbo-1 (May 2019) in this bleeding-edge version:
 - Drop support for Apple's CommonCrypto, it was defunct for many years anyway.
   [magnum; 2021]
 
+- Improved Monero support: Now supports legacy wallets from before 2016 that are
+  not using JSON format yet. [patrickd, magnumripper, solardiz; 2021]
```

Comment:
This should be credited to patrickd only.

Reply:
I'll merge now, and drop the extraneous credits when going over the `NEWS` file prior to the release.

Reply:
Ah ok, in other projects everyone participating in the issue is credited. This might be something to add to the CONTRIBUTE file

Reply:
Fixed in #4735 while at it